pkg/skaffold/config/options.go

```diff
@@ -55,16 +55,16 @@ func (opts *SkaffoldOptions) Labels() map[string]string {
 	labels := map[string]string{}
 
 	if opts.Cleanup {
-		labels["cleanup"] = "true"
+		labels["skaffold-cleanup"] = "true"
```

Comment:
I still think that `skaffold.dev/` would be a better prefix than plain `skaffold-`. Either way will be fine though.

Reply:
How about labels that are already prefixed with `skaffold-` like the ones in `constants.go`?

For instance:

```golang
	TagPolicy:        "skaffold-tag-policy",
	Deployer:         "skaffold-deployer",
	Builder:          "skaffold-builder",
```

Reply:
They, too, should follow the same prefix convention. Whatever it will be :)

Reply:
OK, this is done.